### PR TITLE
Add test and feature for disabling merchant

### PR DESCRIPTION
**Pull request by raymondknguyen**
## Description
Completes User Story #38
Test percentage: 100
Number of Test: 121

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which maintains existing functionality)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## RSpec Results
```

```

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## Description
 Completes User Story #
+Test Percentage:
+Number of Test:
```
Comment (d-atkins) [inline pull_request_template.md:4]: This is already part of the template: at the bottom, in the code block, you can just copy and past your RSpec results

File: db/schema.rb [hunk 2]
```diff
@@ -47,6 +47,7 @@
     t.integer "zip"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.boolean "active", default: true
```
Comment (d-atkins) [inline db/schema.rb:50]: this is a nitpick but I'd make this `active?` just to be consistent with the items table

File: config/routes.rb [hunk 3]
```diff
@@ -45,6 +45,10 @@
   namespace :admin do
     get "/", to: "dashboard#index"
     get "/users", to: "users#index"
+
+    resources :merchants, only: [:index] do
+      patch "/toggle_active", to: "merchants#toggle_active"
```
Comment (d-atkins) [inline config/routes.rb]: I'm thinking this route is a little weird and non restful. I don't really understand resources (and especially not nested ones) but it looks like it's generating a URI of `/admin/merchants/:merchant_id/toggle_active` - I think a patch to just`/admin/merchants/:merchant_id` would be closer to the convention I've seen so far. 

Also, the #toggle_active action is raising a flag for me. I think that could just be an #update. Unless I'm just not seeing it, I don't think admins have access to updating a merchant's information beyond enabling/disabling them, so I think it'd be fine for this functionality to live in #update. 

Those are just my thoughts from the outside looking in so there may be something obvious I'm not considering.

File: app/controllers/admin/merchants_controller.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+class Admin::MerchantsController < ApplicationController
+  def index
+    @merchants = Merchant.all
+  end
+
+  def toggle_active
+    merchant = Merchant.find(params[:merchant_id])
+    merchant.toggle!(:active)
```
Comment (d-atkins) [inline app/controllers/admin/merchants_controller.rb:8]:
```suggestion
    merchant.toggle!(:active?)
```

**Review by d-atkins — CHANGES_REQUESTED** [review]

File: config/routes.rb (hunk 3 above)
Comment (iEv0lv3) [reply config/routes.rb]: I agree with David's comment re: routes

**Review by d-atkins — APPROVED** [review]

**Pull request merged**